doc/user-guide/running.md

```diff
@@ -28,24 +34,46 @@ Download sample notebook:
 curl https://raw.githubusercontent.com/ploomber/ploomber-engine/main/examples/display.ipynb --output running-demo.ipynb
 ```
 
-Import and initialize the notebook client:
+Run the notebook and store the executed version:
+
+```{admonition} Command-line equivalent
+:class: dropdown
+
+`ploomber-engine nb.ipynb output.ipynb`
+```
 
 ```{code-cell} ipython3
-from ploomber_engine.ipython import PloomberClient
+from ploomber_engine import execute_notebook
 
-client = PloomberClient.from_path("running-demo.ipynb")
+nb = execute_notebook("running-demo.ipynb",
+                      output_path="output.ipynb")
 ```
 
-Execute the notebook:
+The function returns a notebook object (same contents as the stored in `output_path`):
```

Comment:
The function returns a notebook object (same contents as ~~the~~ stored in `output_path`):

Reply:
fixed